csrc/preseg_passes/allocation_order_inference.cpp

```diff
@@ -114,6 +172,7 @@ class AllocationOrderInferencer : public IterVisitor {
     return false;
```

Comment:
Now that the if-then branch gets very long, can we `return false` early? I.e.

```
if (found in alloc_order_map_) {
  long code;
  return true;
}
return false;
```
==> 
```
if (not found) {
  return false;
}
long code;
return true;
```

This reduces indentation and gets rid of a branch from readers' minds early.